lib/src/test/java/io/ably/lib/test/realtime/RealtimeCryptoMessageTest.java

```diff
@@ -37,6 +38,12 @@ public void encrypt_message_256() throws IOException, NoSuchAlgorithmException,
 		testEncrypt(testData, "cipher+aes-256-cbc");
 	}
 
+	@Test
+	public void encryptCocoa() throws IOException, NoSuchAlgorithmException, CloneNotSupportedException, AblyException {
```

Comment:
Why are these identified as Cocoa fixtures? Aren't they just common fixtures?

Reply:
Now gone. Any chance you can revisit this PR, please, @paddybyers ? I'm seeking an approval prior to merge. Thanks.